src/tests/test_blindsig.py

```diff
@@ -1,17 +1,16 @@
+# pylint: disable=protected-access
 """
 Test for ECC blind signatures
 """
+from pybitmessage.pyelliptic import ECCBlind, ECCBlindChain, OpenSSL
+
```

Comment:
wrong order

Reply:
There was code quality error popped up saying module level imports should be on top

Reply:
nonsense: this is a library import, why did you place it before the standard library ones and why the for- import is before simple ones? you recently posted a link to pep8 ... so go read it yourself

Reply:
Ok, I reverted the changes
https://github.com/Bitmessage/PyBitmessage/pull/1792/checks?check_run_id=3230496739

Reply:
this branch is out of date, rebase

Reply:
Ok I found the issue, It was not properly rebased. I fixed it now.